students/Mintae_K/lesson02/lesson02_fizz.py

```diff
@@ -0,0 +1,14 @@
+def fizz_buzz():
+    for i in range(100):
+        check3=(i+1) % 3
+        check5=(i+1) % 5
+        if check3 == 0 and not (check5 == 0):
+            #print(check3)
+            print("fizz")
+        elif check5 == 0 and not (check3 == 0):
+            print("buzz")
+        elif check5 == 0 and (check3 == 0):
+            print("fizzbuzz")
```

Comment:
this would flow better if you started from here:
```
if check5 == 0 and check3 == 0:
    print('fizzbuzz')
elif check3 == 0:
    print('fizz')
elif check5 == 0:
    print('buzz')
else:
    print(n)
```
this way your follow if checks are simpler